sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -77,13 +77,13 @@ export default defineComponent({
     const assessmentDataLoaded = (
       applicationStatus: ApplicationStatus,
       noaApprovalStatus: AssessmentStatus,
-      assessmentId: number,
+      currentAssessmentID: number,
     ) => {
       viewOnly.value = !(
         applicationStatus === ApplicationStatus.Assessment &&
         noaApprovalStatus === AssessmentStatus.required
       );
-      currentAssessmentId.value = assessmentId;
+      currentAssessmentId.value = currentAssessmentID;
```

Comment:
Missed in the previous PR but the below also need to be adjusted to remove a warning.
```ts
 assessmentDataLoaded: (
      applicationStatus: ApplicationStatus,
      noaApprovalStatus: AssessmentStatus,
      currentAssessmentId: number,
    ) => {
      return (
        !!applicationStatus && !!noaApprovalStatus && !!currentAssessmentId
      );
    },
```
https://github.com/bcgov/SIMS/blob/9c9cbf39c0cb08c2ca3587a62f2c51b87356c0a3/sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue#L81-L87

Reply:
I've got this now.